.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: build
+      run: |
+        npm install
+        tsc src/extension.ts
+        npx vsce package
+        ls -lt *.vsix
+    - name: release draft
+      env:
+        GITHUB_USER: $GITHUB_ACTOR
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        wget --quiet --output-document=- https://github.com/github/hub/releases/download/v2.12.3/hub-linux-amd64-2.12.3.tgz | tar zx
+        mv hub-linux-* hub
+        ./hub/bin/hub --version
+        REPO=$(basename $GITHUB_REPOSITORY)
+        RELEASE_TAG=$(head -1 CHANGELOG.md | sed 's/^## //')
+        RELEASE_ASSET=zig-$RELEASE_TAG.vsix
+        echo >  RELEASE.md $RELEASE_TAG
+        echo >> RELEASE.md
+        echo >> RELEASE.md Copy the $RELEASE_ASSET file to visual studio code, theia, gitpod, or any other ide that uses vscode extensions.
+        ./hub/bin/hub release create --draft --prerelease --file RELEASE.md --attach $RELEASE_ASSET $RELEASE_TAG
```

Comment:
Is it possible to make this action only run when `CHANGELOG.md` has been modified?

Reply:
I will check. The worst case is that unpublished draft releases are accumulated in chronological order and can be deleted individually. They are not visible to the public.

Reply:
I think I wil back away slightly from full automation at this time. I will refactor to have a separate script that produces a draft release on demand from the programmer.